src/app/modules/shared/components/details-fields/details-fields.component.scss

```diff
@@ -93,3 +93,9 @@ input[type="date"]::-webkit-clear-button {
   border: none;
   margin: 0em;
 }
+
+.valid ::ng-deep .autocomplete-container ::ng-deep .input-container ::ng-deep input  {
```

Comment:
What would be the purpose of this new style?

Reply:
It is the only way to change the style of the "ngx-autocomplete" component without affecting the default style of the component. It's use only for border-color of the input